CMakeLists.txt

```diff
@@ -115,8 +114,7 @@ add_subdirectory(ext/pybind11)
 # Include fmt library
 # =============================================================================
 message(STATUS "INCLUDING FMT")
-add_subdirectory(ext/fmt EXCLUDE_FROM_ALL)
-set_property(TARGET fmt PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
I am unsettled by the forcing of `POSITION_INDEPENDENT_CODE` in a lot of places, coupled by forcing some compilations to be `STATIC`.  What is the reasoning for the latter?

Reply:
I sort of vaguely remember that this came up after including pybind. We had to force `-fPIC`. But I am not entirely sure.

Reply:
Hmm, interesting.  If I were to continue on the cmake cleanup in nmodl, I'd be tempted to see if I can remove the explicit STATICs, and once they are gone, see if this can be changed.  However, I suspect you're right.